### DISCUSSION TITLE
Why use pick function to get request params etc?

### DISCUSSION THREAD

**Discussion by saqib-ahmed1** [hagopj13/node-express-boilerplate#248] [Q&A]
I was wondering what is the added benefit of using `pick(req.param, ['filter'])` over directly accessing the variable `req.param.filter`. Is there a security benefit of using it like preventing SQL injections etc?

**Comment by amadebusuyi** [comment]
@saqib-ahmed1 There are no security benefits, just makes it easier to compile the object for you, particularly when you are filtering multiple fields.

Just in case you're still wondering.
